NWEEI/NWEEI/Views/Article/ByCategory.cshtml

```diff
@@ -60,17 +61,40 @@ else
     <h1>@Model[0].Category.Name Articles</h1>
 
     <div class="articles container">
-        @foreach (var item in Model)
+        @foreach (var article in Model)
         {
-            <div class="article-tile-container">
-                <div class="article-tile-content">
-                    <a asp-action="Details" asp-route-id="@item.ArticleID">
-                        <div class="article-tile-title">@item.Title</div>
-                        @*<div class="article-tile-preview">@item.Body.Substring(0, 150)...</div>*@
-                        <div class="article-tile-date">@item.PublishDate.ToShortDateString()</div>
-                    </a>
+            <a asp-action="Details" asp-route-id="@article.ArticleID">
+                <div class="article-tile-container">
+                    <div class="article-tile-content">
+
+                        <div class="row">
+                            <div class="col-10">
+                                <!-- article title -->
+                                <div class="article-tile-title">@article.Title</div>
+                            </div>
+
+                            <div class="col-2">
+                                <!-- article date -->
+                                <div class="article-tile-date">@article.PublishDate.ToShortDateString()</div>
+                            </div>
+                        </div>
+
+                        <!-- article body preview -->
+                        <div class="article-tile-preview">
+                            @if (article.Body.Length < 150)
+                            {
+                                @Regex.Replace(article.Body, "<.*?>", string.Empty)
+                            }
+                            else
+                            {
+                                @Regex.Replace(article.Body.Substring(0, 150), "<.*?>", string.Empty)
+                                @Html.Raw("<span>...</span>")
+                            }
```

Comment:
I like that the article cuts off after 150 characters, it make sense not to show the whole thing (unless, of course, it fits). I have noticed one small thing that we might want to fix, and I created a [bug sub-task in Jira (CAP-320)](https://jarthur.atlassian.net/browse/CAP-320) to track it.